tests/SphericalCollapse.in

```diff
@@ -23,7 +23,7 @@ amr.grid_eff        = 0.7   # default
 hydro.reconstruction_order = 3  # PPM
 cfl = 0.25
 max_timesteps = 50000
-stop_time = 0.15
+stop_time = 660
```

Comment:
This will make the number of free-fall times the same as before, but will also make the number of sound-crossing times a factor of 10^7 larger than before. I think the best solution is to just set Gconst_ = 1 for only this test problem.

Reply:
I agree that it will make the cloud effectively colder, and that one would need to increase the sound speed as well to keep all the dimensionless numbers the same. It's fine to just set G = 1 if that is easier.